### PR TITLE
Fixes some mechs still having EMP armor

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Missed a couple in #16605

# Changelog

:cl:    
bugfix: Certain mechs that had EMP resistance no longer have any
/:cl:

### CODE REVIEW

**Comment by Maia-J** [comment]
The DS one should still keep some, they are meant to be unfair

**Comment by ynot01** [comment]
> The DS one should still keep some, they are meant to be unfair

thats not a deathsquad mech thats a meme adminspawn mech that gets a clamp for a weapon

the deathsquad mechs already have 0 energy armor

**Comment by Mqiib** [comment]
Let the dark gygax keep its energy armor, it honestly kinda needs it.

File: code/game/mecha/working/ripley.dm [hunk 1]
```diff
@@ -118,7 +118,7 @@
 
 /obj/mecha/working/ripley/deathripley/real
 	desc = "OH SHIT IT'S THE DEATHSQUAD WE'RE ALL GONNA DIE. FOR REAL"
-	armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 80, BOMB = 90, BIO = 100, RAD = 100, FIRE = 100, ACID = 100)
+	armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 0, BOMB = 90, BIO = 100, RAD = 100, FIRE = 100, ACID = 100)
```
Comment (MajManatee) [inline code/game/mecha/working/ripley.dm:121]: Let the poor evil ripley have its armor

**Review by Moltijoe — APPROVED** [review]
number change easy
DS ripley should be emp immune for the lulz

**Pull request merged**